docker/APIDockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12-alpine
+COPY . .
+RUN yarn install --production
+RUN npx sequelize-cli db:migrate
```

Comment:
do we need this for the dockerfile? wouldn't this be part of a db integration?

Getting

```
Step 4/6 : RUN npx sequelize-cli db:migrate
 ---> Running in 33e0711d9a27
npx: installed 80 in 7.338s

Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.6]

Loaded configuration file "src/server/db/config/config.js".
Using environment "development".
ERROR: connect ECONNREFUSED 127.0.0.1:5432
The command '/bin/sh -c npx sequelize-cli db:migrate' returned a non-zero code: 1
```
I don't think we care about having a db during a build unless we really want to test integration during build.